app/controllers/admin/dashboard_controller.rb

```diff
@@ -41,18 +35,38 @@ def prep_projects_grids_for_division_users
 
     @people_grids = {}
     @people.each do |person|
-      @people_grids[person] = initialize_grid(
-        person.active_agent_projects.order(created_at: :desc, updated_at: :desc),
-        include: [:primary_agent, :secondary_agent],
-        per_page: 5,
-        name: "projects_person_#{person.id}",
-        enable_export_to_csv: false
-      )
+      projects = person.active_agent_projects
+      @people_grids[person] = initialize_wice_grid(projects, person, 5)
     end
   end
 
   def prep_logs
     @context = "dashboard"
     @logs = ProjectLog.in_division(selected_division).where(agent_id: @person.id).by_date.page(1).per(10)
   end
+
+  private
+
+  def initialize_wice_grid(projects, person, page)
+    initialize_grid(
+        projects,
+        include: [:primary_agent, :secondary_agent],
+        per_page: page,
+        order: 'projects.order_by_agent',
+        custom_order: {
+            'projects.order_by_agent' => "case when projects.primary_agent_id = #{person.id} and projects.status_value = 'active' and projects.type = 'Loan' then 8
```

Comment:
the logic can be moved to the model but I can't shorten the `pai` and `sai`. Well not without typing them out first.

The `=>` here is the syntax being supported by the gem itself and also since I have a dot syntax `projects.order_by_agent` and with the who sql-ish-ness, having the new JSON style won't work here